### ISSUE TITLE
Cross trim with T5 and T6 no effect

### ISSUE THREAD

**Issue by phaedra-tm** [FrSkyRC/ETHOS-Feedback-Community#2119]
I imported a model from my X20S, where I had cross-trim configured for elevator (I used the throttle trim for elevator).
When I change the elevator stick trim to T5 or T6, it doesn't do anything.
Strange things was also that as long as I only changed the elevator stick trim to T5, but left throttle stick trim to "elevator", the throttle trim still moved the elevator.
Firmware 1.4.3

**Comment by mawzthefinn** [comment]
I just tried the same thing on my X18.

I can confirm that T5 is not doing anything in this situation, but I don't get elevator trim working either.

**Comment by phaedra-tm** [comment]
Forgot to mention: I had this issue on my Tandem XE

**Comment by bsongis-frsky** [comment]
![image](https://user-images.githubusercontent.com/50834778/204567432-70f8f410-4cae-466f-823c-ee51ce09fc98.png)

**Comment by bsongis-frsky** [comment]
Test KO. Opening again to avoid I forgive it

**Comment by phaedra-tm** [comment]
Maybe related to this, but today I had 2 occurrences today where I shut down the radio with cross trim between elevator and throttle setup (stick elevator -> trim throttle).  When I powered back on,  cross trim was set to stick elevator -> T5.  Set it back to stick elevator -> trim throttle and did a flight (worked OK).  Shut down radio after flight, powered back on, cross trim was back at stick elevator -> T5.
When I tried this a third time, my settings remained at stick elevator -> trim throttle.  Hard to reproduce consistently.
